src/components/SecondaryPanes/Expressions.css

```diff
@@ -43,8 +43,9 @@
 }
 
 .expression-container > .tree {
-  overflow: hidden;
+  overflow: auto;
```

Comment:
Hi @wldcordeiro 
`overflow: auto` seems to cause the scroll bar to appear when you hove over an element. Reverting overflow to `hidden` would fix this issue.

<img width="1195" alt="screen shot 2017-02-09 at 1 54 14 pm" src="https://cloud.githubusercontent.com/assets/4562118/22771028/67515b5e-eecf-11e6-95fc-85e294f2df04.png">